jsk_pcl_ros/test/pointcloud_database_server.test

```diff
@@ -0,0 +1,17 @@
+<launch>
+  <include file="$(find jsk_pcl_ros)/sample/sample_pointcloud_database_server.launch">
+    <env name="DISPLAY" value="" />
```

Comment:
`sample_bing.test` is sample program, so it launches gui programs [here](https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception%2Fsample%2Fsample_bing.launch#L23-L37)
Setting `DISPLAY` env as empty is to prevent image_view to launch, but your program has no GUI program, so you don't need that line.